R/format_geoparquet.R

```diff
@@ -0,0 +1,16 @@
+format_geoparquet <- tar_format(
+  read = function(path) {
+    arrow::read_parquet(path) |>
+      dplyr::mutate(dplyr::across(wk::is_handleable, wk::as_wkb))
+  },
+  write = function(object, path) {
+    arrow::write_parquet(object, path)
+  },
+  marshal = function(object) {
+    arrow::as_arrow_table(object)
```

Comment:
Possibly here too? EDIT: as above

```suggestion
    arrow::as_arrow_table(as.data.frame(object, geom = "WKT"))
```